docs/conf.py

```diff
@@ -212,14 +212,34 @@
         # A RTD thing to avoid a CSRF block when docs are using a custom domain
         hoverxref_api_host = "/_"
 
-hoverxref_tooltip_maxwidth = 600  # RTD main window is 696px
-hoverxref_auto_ref = True
-hoverxref_mathjax = True
-# hoverxref has to be applied to these
+hoverxref_auto_ref = False
```

Comment:
I had to disable this, I can't get it to not work on gallery examples.

Either I am missing a config value or we need to consider a patch upstream.

Reply:
I think if we could change: https://github.com/readthedocs/sphinx-hoverxref/blob/b4b5cf4ae95cbb78151509ed46b5b0c5f821ec58/hoverxref/domains.py#L60 to string match instead, we could add the generic gallery text.